src/utils/options.js

```diff
@@ -10,6 +10,15 @@ const messageDescription = `Specifies a source for providing a message to the co
 	- --message stdin
 `;
 
+const secondPublicKeyDescription = `Specifies a source for providing a second public key to the command. The second public key must be provided via this option. Sources must be one of \`file\` or \`stdin\`. In the case of \`file\`, a corresponding identifier must also be provided.
```

Comment:
If this option is specific to the command, it should probably go in the command file rather than here.